src/toolbar/toolbar.jsx

```diff
@@ -46,21 +67,36 @@ const Toolbar = React.createClass({
     return this.state.muiTheme.toolbar;
   },
 
+  getSpacing() {
```

Comment:
My thoughts exactly!! But this is somehow a bit global. I wanted to keep consistent with rest and propose this later on. If you agree, and follow the same path, we'll enforce some code styles as we migrate the docs and their corresponding components.